src/Cake.Docker/Cake.Docker.csproj

```diff
@@ -1,12 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0;netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
```

Comment:
Drop support for `netstandard2.0` to align with Cake 2.0.
```suggestion
    <TargetFrameworks>netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
```

Reply:
Done